include/rocksdb/convenience.h

```diff
@@ -458,17 +409,11 @@ Status GetStringFromCompressionType(std::string* compression_str,
 
 std::vector<CompressionType> GetSupportedCompressions();
 
-Status GetBlockBasedTableOptionsFromString(
-    const BlockBasedTableOptions& table_options, const std::string& opts_str,
-    BlockBasedTableOptions* new_table_options);
 Status GetBlockBasedTableOptionsFromString(
     const ConfigOptions& config_options,
     const BlockBasedTableOptions& table_options, const std::string& opts_str,
     BlockBasedTableOptions* new_table_options);
 
-Status GetPlainTableOptionsFromString(const PlainTableOptions& table_options,
-                                      const std::string& opts_str,
-                                      PlainTableOptions* new_table_options);
```

Comment:
Note to reviewer: These 3 functions are not explicitly marked as deprecated like  `GetColumnFamilyOptionsFromString` but I don't see reason for not removing them, consider their similarities.